package.json

```diff
@@ -13,14 +13,16 @@
     "url": "http://github.com/stellar/account-viewer.git"
   },
   "dependencies": {
+    "@ledgerhq/hw-app-str": "4.2.0",
+    "@ledgerhq/hw-transport-u2f": "4.2.0",
+    "babel-runtime": "^6.26.0",
```

Comment:
Is `babel-runtime` needed anywhere?

Reply:
Yes, I don't know why it's not a transitive dependency of the ledger library but without it it will not run.

Reply:
We fell into similar thing too and it caused a lot of problems for users: https://github.com/stellar/js-xdr/pull/9 (now fixed). I'm going to merge this but let Ledger team know about this.